api/planmill1_5.raml

```diff
@@ -1531,6 +1532,50 @@ resourceTypes:
     type:
       getDeleteCollectionItem:
         exampleCollectionItem: Contents of file
+/resources:
+  description: View details of resources ( schedule + assignment ).  
```

Comment:
View details of resources ( schedule + assignment ).  -> View details of resources ( resource = assignment + schedule).